analytic_engine/src/row_iter/merge.rs

```diff
@@ -127,7 +126,7 @@ impl<'a> MergeBuilder<'a> {
             config,
             sampling_mem: None,
             memtables: Vec::new(),
-            ssts: vec![Vec::new(); MAX_LEVEL],
+            ssts: vec![Vec::new(); Level::TOTAL_NUM],
```

Comment:
It seems a bit strange, return the `TOTAL_NUM` from the single `Level`?
Maybe it should belong to something like levels?

Reply:
If you are uncomfortable, I can move it out of Level, and define a `SST_TOTAL_LEVEL` in file.rs directly.

Reply:
It sounds better.